spyder/app/mainwindow.py

```diff
@@ -2720,6 +2745,36 @@ def show_tour(self, index):
         self.tour.set_tour(index, frames, self)
         self.tour.start_tour()
 
+    # ---- Global File Switcher
+    def open_fileswitcher_dlg(self):
+        """Open file list management dialog box."""
+        if self.fileswitcher_dlg is not None and \
+          self.fileswitcher_dlg.is_visible:
+            self.fileswitcher_dlg.hide()
+            self.fileswitcher_dlg.is_visible = False
+            return
+        self.fileswitcher_dlg.setup()
+        self.fileswitcher_dlg.show()
+        self.fileswitcher_dlg.is_visible = True
+
+    def open_symbolfinder_dlg(self):
+        """Open symbol list management dialog box."""
+        self.open_fileswitcher_dlg()
+        self.fileswitcher_dlg.set_search_text('@')
+
+    def add_to_fileswitcher(self, plugin, tabs, data):
+        """Add a plugin to the File Switcher."""
+        if self.fileswitcher_dlg == None:

```

Comment:
The convention when comparing with `None` is:

    if self.fileswitcher_dlg is None:

Please change it to be this way.